dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1707,6 +1707,15 @@ public void deleteAllVersionsandBackup(List<Contentlet> contentlets, User user,
         }
         deleteBinaryFiles(contentletsVersion,null);
 
+        try {
+            String bla = null;
+            bla = "fffff";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "fffff" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)